src/components/Button/Button.tsx

```diff
@@ -7,8 +11,10 @@ interface IButtonProps {
 }
 
 const Button: FC<IButtonProps> = ({ className, disabled, children}) => {
+  const button = cn('Button')
```

Comment:
Ты всё верно написал, но я пришёл к тому, что лучше писать в стиле `cnClassName`:
```suggestion
  const cnButton = cn('Button')
```

Так не будет пересечений с обычными переменными.